api/swagger/swagger.yaml

```diff
@@ -519,6 +519,9 @@ definitions:
       has_published_results:
         type: boolean
         description: Trial has its results published (true/false)
+      is_registered:
+        type: boolean
+        description: Whether the source is a non registry or not
```

Comment:
The description feels too focused on the implementation details. Something like `Whether the trial is registered in a clinical trial registry` is clearer IMHO.